CHANGELOG.md

```diff
@@ -10,6 +10,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Generic type support to `ListArtifact`.
 - Iteration support to `ListArtifact`.
 
+### Changed
+- **BREAKING**: Split `BaseExtractionEngine.extract` into `extract` and `extract_artifacts` for consistency with `BaseSummaryEngine`.
+- **BREAKING**: `BaseExtractionEngine` no longer catches exceptions and returns `ErrorArtifact`s.
```

Comment:
Parallel discussion: we really need to figure out when and why we use `ListArtifact`s and `ErrorArtifact`s over `list`s and `Exception`s

Reply:
I think it should be done nearly globally, but it saves us from writing code like this:

```python
if isinstance(result, ErrorArtifact):
    print(result.to_text())
else:
    for row in result.value:
        print(row.to_text())
```

Maybe we should agree on this pattern before implementing anywhere.